ToyBox/classes/MainUI/BlueprintAction.cs

```diff
@@ -27,9 +28,9 @@
 
 namespace ToyBox {
     public abstract class BlueprintAction {
-        public delegate void Perform(SimpleBlueprint bp, UnitEntityData ch = null, int count = 1);
+        public delegate void Perform(SimpleBlueprint bp, UnitEntityData ch = null, int count = 1, int listValue = 0);
```

Comment:
please consider a more descriptive name for listValue.  Also a comment would be nice.